internal/rest/resources/certificates.go

```diff
@@ -82,6 +82,11 @@ func clusterCertificatesPut(s state.State, r *http.Request) response.Response {
 	var certificateDir string
 	if certificateName == string(types.ClusterCertificateName) {
 		certificateDir = s.FileSystem().StateDir
+	} else if certificateName == string(types.ServerCertificateName) {
```

Comment:
Mh, in this case do we have to stop doing the cluster notification a few lines above? I guess otherwise we will update the server certificate on all nodes.

Before this wasn't an issue because if the cert name wasn't `cluster` it was checking in the `certificates` dir.

Reply:
The cluster notification has a check that requires the database to be online. It will actually catch more states than the check in this block which explicitly wants `StatusNotReady`

Reply:
https://github.com/canonical/microcluster/blob/6375e4d511c6458c6011e0a7a83057742bd07719/internal/rest/resources/certificates.go#L47-L53

Reply:
True, I haven't seen the `s.Database().IsOpen(r.Context())` check. Thx.